vignettes/howto-dm-df.Rmd

```diff
@@ -133,7 +134,8 @@ flights_dm_all_keys <-
   flights_dm_only_pks %>%
   dm_add_fk(table = flights, columns = tailnum, ref_table = planes) %>%
   dm_add_fk(flights, carrier, airlines) %>%
-  dm_add_fk(flights, origin, airports)
+  dm_add_fk(flights, origin, airports) %>%
+  dm_add_fk(flights, c(origin, time_hour), weather)
```

Comment:
Do we want to mention `weather` in the text above, and that it's a compound key?

Reply:
> Note that {dm} functions work with both named and positional argument specification, and compound keys can be specified using a vector argument.

I had added this sentence below this code chunk. I thought that'd be enough. WDYT?